docs/content/tutorial/setup.mdx

```diff
@@ -9,25 +9,12 @@ To complete this tutorial, you'll need:
 
 - **To install Python and pip.** This tutorial assumes that you have some familiarity with Python, but you should be able to follow along even if you're coming from a different programming language. To check that Python and `pip` (Python's package manager) are already installed in your environment or install them, follow the instructions [here](https://packaging.python.org/guides/installing-using-pip-and-virtual-environments/). <DagsterVersion />
 
-- To install Dagster, Dagit, and the packages you'll be using during this tutorial, run the following command from your terminal:
+- To install Dagster, run the following command from your terminal:
```

Comment:
```suggestion
- **To install Dagster**. Run the following command from your terminal:
```

Sorry to be nitpicky about this! This makes it consistent with the first list item.